docs/EN_US/HPCCSystemAdmin/SA-Mods/WUTool.xml

```diff
@@ -82,6 +82,23 @@
           <entry>Restore from xml files. [INCLUDEFILES=1]</entry>
         </row>
 
+        <row>
+          <entry>importzap</entry>
+
+          <entry>Imports ZAP report to be able to recreate a workunit and
+          replicate the reported issue. importzap requires the following
+          parameters. <para>&lt;zapreport-filename&gt;
+          &lt;output-helper-directory&gt;
+          [&lt;zapreport-password&gt;]</para></entry>
```

Comment:
zapreport-password is optional. Also, it is unclear what **output-helper-directory** is.